packages/loot-core/src/server/aql/compiler.ts

```diff
@@ -694,7 +694,7 @@ const compileOp = saveStack('op', (state, fieldRef, opData) => {
     }
     case '$ne': {
       if (castInput(state, rhs, lhs.type).type === 'null') {
-        return `${val(state, lhs)} IS NULL`;
```

Comment:
This seemed both wrong and a critical piece of code, so a second pair of eyes here would be good?

Reply:
This seems correct